source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -465,6 +467,9 @@ void Filter::onGrpcError(Grpc::Status::GrpcStatus status) {
 
 void Filter::onGrpcClose() {
   ENVOY_LOG(debug, "Received gRPC stream close");
+  if (processing_complete_) {
+    return;
+  }
```

Comment:
How is this related to disabling timers?

Reply:
This was a belt-and-suspenders check that it turns out we don't actually need, so I'm going to take it out.